js/components/controlBar.js

```diff
@@ -992,7 +992,9 @@ var ControlBar = createReactClass({
       'oo-flex-end': this.props.audioOnly && Utils.isIos()
     });
 
-    return (
+    return (this.props.skinConfig &&
```

Comment:
isn't it better to return `null` at the beginning of render?

Reply:
\+
thank you for the notice!